ring/func/base.py

```diff
@@ -638,7 +639,7 @@ class RingRope(RopeCore):
             def __init__(self, *args, **kwargs):
                 super(RingRope, self).__init__(*args, **kwargs)
                 self.user_interface = self.user_interface_class(self)
-                self.storage = self.storage_class(self, storage_backend)
+                self.storage = self.storage_class(self, storage_backend, maxsize)
```

Comment:
Does redis_hash failure related to this change?

Reply:
Yes, `__init__` is overridden in `RedisHashStorage`. I'm looking at right now and seems there are some workarounds to get back to work but not sure which one will be the best one.
The problem is that `BaseStorage` has a __init__ and `ExpirableDictStorage` & `PersistentDictStorage` adheres it but `RedisHashStorage` overrides it.

Can you tell me your opinion?